.github/workflows/publishRestApiClient.yml

```diff
@@ -15,7 +15,14 @@ jobs:
             - name: Dereference OpenAPI doc
               run: npx json-dereference-cli json-dereference -s repo/rest-api/src/RouteHandlers/openapi.json -o openapi-dereferenced.json
             - name: Define next package version
-              run: echo "VERSION=$(npx semver $(npm view wb-rest-api-client version) --increment minor)" >> $GITHUB_ENV
+              run:  |
+                  CURRENT_VERSION=$(npm view wb-rest-api-client version || echo "null")
+                  if [ "$CURRENT_VERSION" = "null" ]; then
+                     VERSION="0.1.0"
+                  else
+                     VERSION=$(npx semver "$CURRENT_VERSION" --increment minor)
+                  fi
+                  echo "VERSION=$VERSION" >> $GITHUB_ENV
```

Comment:
Woohoo nice!! @dkoushha  I think we can simplify that a bit to just:

```bash
echo "VERSION=$(npx semver $(npm view wb-rest-api-client version 2> /dev/null || echo 0.0.0) --increment minor)" >> $GITHUB_ENV
```

You can try that out locally 
`echo $(npx semver $(npm view wb-rest-api-client version 2> /dev/null || echo 0.0.0) --increment minor)` prints 0.10.0 as it should vs 

`echo $(npx semver $(npm view @wmde/wikibase-rest-api version 2> /dev/null || echo 0.0.0) --increment minor)` which prints 0.1.0 :)